src/aliceVision/matchingImageCollection/GeometricFilter.cpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2022 AliceVision contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "GeometricFilter.hpp"
+
+namespace aliceVision {
+namespace matchingImageCollection {
+
+void removePoorlyOverlappingImagePairs(PairwiseMatches& geometricMatches,
+                                       const PairwiseMatches& putativeMatches,
+                                       float minimumRatio,
+                                       std::size_t minimumGeometricCount)
+{
+    std::vector<PairwiseMatches::key_type> toRemoveVec;
+    for (auto iterMap = geometricMatches.begin(); iterMap != geometricMatches.end(); ++iterMap)
```

Comment:
I think it can be `const`
```suggestion
    for (const auto iterMap = geometricMatches.begin(); iterMap != geometricMatches.end(); ++iterMap)
```

Reply:
Also it should be possible to use a range loop here, no?
```suggestion
    for (const auto& mapElement : geometricMatches)
```

Reply:
Fixed, thanks.